cmd/kubectl_token.go

```diff
@@ -372,7 +414,8 @@ func samlAuth(input *LoginInput, tlsConfig *tls.Config) (managementClient.Token,
 			client = &http.Client{Transport: tr, Timeout: 150 * time.Second}
 			res, err = client.Do(req)
 			if err != nil {
-				return token, err
+				// log error and use the token if login succeeds
+				customPrint(fmt.Sprintf("DeleteToken: %v", err))
```

Comment:
Is this right ? Seems kinda weird message. Looks like you are telling user to do something ?

Reply:
Sorry it was to be fmt.Errorf...all other msgs are fmt.Errors with what action on token it's failing, and err is the msg